app/components/LeftPaneEntries/EntryItem/index.tsx

```diff
@@ -177,9 +184,8 @@ function EntryItem(props: EntryItemProps) {
                         <ExcerptInput
                             value={excerpt}
                             image={entryImage}
-                            imageRaw={imageRaw}
```

Comment:
We shouldn't replace imageRaw, if we need LeadAttachmentImage, lets put it separately life we are doing for entryAttachment.

Reply:
Let's not add extra props for ExcerptInput for each special case if we can just use imageRaw.